src/Compilers/CSharp/Test/Symbol/Symbols/InterfaceImplementationTests.cs

```diff
@@ -2711,5 +2711,53 @@ static void Main()
             var implementingMember = derivedType.FindImplementationForInterfaceMember(interfaceMember);
             Assert.Equal(expectedImplementingMember, implementingMember.ToTestDisplayString());
         }
+
+        [Fact]
+        [WorkItem(50713, "https://github.com/dotnet/roslyn/issues/50713")]
+        public void Issue50713_1()
+        {
+            var text1 = @"
+public interface I1
+{
+    void M();
+}
+
+public interface I2 : I1
+{
+    new void M();
+}
+";
+
+            var comp1 = CreateCompilation(text1);
+            comp1.VerifyDiagnostics();
+
+            var i1M = comp1.GetMember("I1.M");
+            var i2 = comp1.GetMember<NamedTypeSymbol>("I2");
+            Assert.Null(i2.FindImplementationForInterfaceMember(i1M));
```

Comment:
The doc comment for this member says:

> Returns the corresponding symbol in this type or a base type that implements interfaceMember

Given that I would expect that this returns a non-null value as the base type, or base interface, for `i2` does implement this member. Is the doc comment wrong here or am I misreading it somehow?

Reply:
> Given that I would expect that this returns a non-null value as the base type, or base interface, for i2 does implement this member. Is the doc comment wrong here or am I misreading it somehow? 

A declaration of a member in an interface is not an implementation. Also, even if a method was implemented by an interface (let's say written in C#), VB doesn't recognize those as implementations and would require a class to implement them anyway. 

---
In reply to: [563892891](https://github.com/dotnet/roslyn/pull/50722#discussion_r563892891) [](ancestors = 563892891)